src/Constants.ts

```diff
@@ -671,3 +681,5 @@ export const BROKER_WS_HTTP_ERR_CODE_TO_MSG: {[key: number]: string} = {
     401: `Authentication is failed, check your Thundra debugger authentication token.`,
     409: `Another session with the same session name exists, connection closed.`,
 };
+
+export const THUNDRA_TRACE_LINK_KEY = '_thundra';
```

Comment:
I think, `THUNDRA_TRACE_KEY` is better name as in the future there might be additional data under `_thundra` key, not just trace link